src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -249,5 +249,35 @@ export function getAuthenticatorDisplayName(remediation) {
   return remediation.relatesTo?.value?.displayName;
 }
 
+export function getVerifyAuthenticatorAriaLabel(authenticatorModel) {
+  // For Custom App: label is "Get a push notification", description is the app name
+  // For Okta Verify: label is "Use Okta FastPass" / "Get a push notification" / "Enter a code",
+  //   description is "Okta Verify"
+  // ARIA label should be eg. "Select Okta Verify. Use Okta FastPass"
+  const isNameInDescription = [
+    AUTHENTICATOR_KEY.CUSTOM_APP,
+    AUTHENTICATOR_KEY.OV,
+  ].includes(authenticatorModel.get('authenticatorKey'));
+  let name = authenticatorModel.get('label');
+  let description = authenticatorModel.get('description');
+  if (description && isNameInDescription) {
+    [name, description] = [description, name];
+  }
+  return [
+    [
+      loc('oie.verify.authenticator.button.text', 'login'),
+      name,
+    ].join(' '),
+    description,
+  ].filter(v => v).join('. ');
```

Comment:
I think this is fine for now, especially given the priority of the ticket, but can you create a ticket to create dedicated i18n keys for these buttons? Usually it is not recommended to concatenate translated strings due to the language context differences.